tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -98,6 +99,7 @@ options:
 	shim-v2
 	tdvf
 	virtiofsd
+	boot-image-se
```

Comment:
I think this is supposed to be alphabetized.

Reply:
Yep, thanks.